image-definitions/kubeflow/components/example-notebook-servers/Makefile

```diff
@@ -0,0 +1,49 @@
+TAG ?= $(shell git describe --tags --always --dirty)
```

Comment:
Removed building/publishing of notebook examples that we are not planning to support.